src/main/java/net/atos/zac/zoeken/IndexeerService.java

```diff
@@ -259,15 +258,15 @@ private void createEntity(final String id, final ZoekObjectType type, final Inde
     }
 
     private boolean isHerindexeren(final ZoekObjectType type) {
-        return countEntities(type, IndexStatus.INDEXED) > 0;
+        return countZoekIndexEntities(type, IndexStatus.INDEXED) > 0;
     }
 
-    private void processSolrIndex(final ZoekObjectType type) {
+    private void addExistingSOLREntitiesAsZoekIndexEntitiesToBeRemoved(final ZoekObjectType type) {
```

Comment:
Deze naam is wel heel technisch voor een private functie, SOLR zou Solr moeten zijn. herindexeerReadSolrIndex ofzo..

Reply:
Naam is wel technisch maar beschrijft wel wat ie doet.